state/models.go

```diff
@@ -241,6 +241,7 @@ type ExecutionRequestCommon struct {
 	IdempotenceKey        *string         `json:"idempotence_key,omitempty"`
 	Arch                  *string         `json:"arch,omitempty"`
 	Labels                *Labels         `json:"labels,omitempty"`
+	ServiceAccount        *string         `json:"service_account,omitempty"`
```

Comment:
The Koco server will have a mapping between DAGs to ServiceAccounts, and after getting Flotilla's requests with a DAG_id, the corresponding service account will be returned which will be used to launch eks pods. That is, this will NOT be passed over from Futura Operator.

Reply:
Correct